frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -167,13 +173,32 @@ export const podRingLabel = (
         subTitle = `scaling to ${desiredPodCount}`;
       } else {
         title = currentPodCount;
-        subTitle = pluralizeString(currentPodCount, 'pod');
+        subTitle = podKindString(currentPodCount);
       }
       return {
         title,
         subTitle,
         titleComponent: getTitleComponent(),
       };
+    case PodModel.kind:
+    case JobModel.kind:
+      currentPodCount = 1;
+      title = currentPodCount;
+      subTitle = podKindString(currentPodCount);
+      return {
+        title,
+        subTitle,
+        titleComponent: getTitleComponent(),
+      };
+    case CronJobModel.kind:
+      currentPodCount = pods.length;
+      title = `${currentPodCount}`;
+      subTitle = podKindString(currentPodCount);
+      return {
+        title,
+        subTitle,
+        titleComponent: getTitleComponent(),
+      };
```

Comment:
nit: can we have it as below or `currentPodCount` being used at other place?
```suggestion
      return {
        title : pods.length,
        subTitle: podKindString(currentPodCount),
        titleComponent: getTitleComponent(),
      };
```

Reply:
It _has_ to be a string or a zero value will be ignored (there is a check for `!title` in the underlying components).